ext/openssl/openssl.c

```diff
@@ -6801,32 +6802,34 @@ PHP_FUNCTION(openssl_random_pseudo_bytes)
 	zend_string *buffer = NULL;
 	zval *zstrong_result_returned = NULL;
 
-	if (zend_parse_parameters(ZEND_NUM_ARGS(), "l|z/", &buffer_length, &zstrong_result_returned) == FAILURE) {
-		return;
-	}
+	ZEND_PARSE_PARAMETERS_START(1, 2)
```

Comment:
I don't think this needs to be changed to use macro syntax - all calls to OpenSSL are quite heavy usually so it doesn't do any visible improvements. That's why it's not used anywhere else in openssl.c.

Reply:
That's fair. I'll revert this back to the zpp function after voting ends. :)